spec/acceptance/nodesets/ubuntu-16-docker.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  ubuntu-16-docker:
+    roles:
+      - master
+    platform: ubuntu-1404-x86_64
+    image: ubuntu:14.04
+    docker_cmd: '["/sbin/init"]'
```

Comment:
I think this is the main problem with Docker-based Ubuntu 16.04 tests at the moment. Need to remove this override. Whatever command we pass to Beaker here, it needs to start sshd listening for SSH connections. I don't think that's working with this command on Ubuntu. SSH never connects. If I remove this command and use the default to start sshd, it works.